core/frontend/src/views/ExtensionManagerView.vue

```diff
@@ -182,20 +220,55 @@ export default Vue.extend({
       status_text: '',
       log_output: null as null | string,
       show_log: false,
+      metrics: {} as any,
+      metrics_interval: 0,
       settings,
     }
   },
   mounted() {
     this.fetchManifest()
     this.fetchInstalledExtensions()
     this.fetchRunningContainers()
+    this.fetchMetrics()
+    this.metrics_interval = setInterval(this.fetchMetrics, 30000)
+  },
+  destroyed() {
+    clearInterval(this.metrics_interval)
   },
   methods: {
     getContainer(extension: InstalledExtensionData): RunningContainer[] | undefined {
       return this.running_containers.filter(
         (container) => container.image === `${extension.name}:${extension.tag}`,
       )
     },
+    async fetchMetrics(): Promise<void> {
+      await back_axios({
+        method: 'get',
+        url: `${API_URL}/stats`,
+        timeout: 20000,
+      })
+        .then((response) => {
+          this.metrics = response.data
+        })
+        .catch((error) => {
+          notifier.pushBackError('EXTENSIONS_METRICS_FETCH_FAIL', error)
+        })
+    },
+    getCpuUsage(extension: InstalledExtensionData): number {
+      const name = this.getContainerName(extension)?.replace('/', '')
+      if (!name) {
+        return NaN
+      }
+
+      return this.metrics[name]?.cpu
```

Comment:
```suggestion
      return this.metrics[name]?.cpu ?? NaN
```

Reply:
doesnt seem to work 
<img width="665" alt="Screenshot 2022-11-18 at 16 08 46" src="https://user-images.githubusercontent.com/4013804/202783746-2dff66d5-d451-43b1-b1b9-11c9614d317f.png">